src/v2/Apps/Purchase/Components/OrderRow.tsx

```diff
@@ -0,0 +1,284 @@
+              </Text>
+            </Link>
+          )}
+          {orderIsInactive && (
+            <Text variant="text" letterSpacing="tight">
+              {artwork?.artist_names}
+            </Text>
+          )}
+          <Text variant="text" color="black60" letterSpacing="tight">
+            {artwork?.partner?.name}
+          </Text>
+          <Text variant="text" color="black60" letterSpacing="tight">
+            {orderCreatedAt.toLocaleString(DateTime.DATE_SHORT)}
+          </Text>
+        </Flex>
+        <Flex flexDirection="column" alignItems="flex-end">
+          <Text
+            variant="text"
+            color="black60"
+            letterSpacing="tight"
+            style={{ textTransform: "capitalize" }}
+          >
+            {order.buyerTotal}
+          </Text>
+          <Text
+            variant="text"
+            color={getOrderColor(order.state.toLowerCase())}
+            letterSpacing="tight"
+            style={{ textTransform: "capitalize" }}
+          >
+            {order.state.toLowerCase()}
+          </Text>
+        </Flex>
+      </Flex>
+      {props.hasDivider && <Separator />}
+    </Box>
+  )
+
+  const SMOrderRow = (
+    <BorderBox my={2} mx="40px" p={0} flexDirection="column">
+      <Flex
+        bg="black5"
+        width="100%"
+        justifyContent="space-between"
+        alignItems="center"
+        p={2}
+      >
+        <Text variant="text">
+          {" "}
+          {orderCreatedAt.toLocaleString(DateTime.DATE_MED)}
+        </Text>
+        <Flex alignItems="center">
+          {getOrderIcon(order.state.toLowerCase())}
+          <Text
+            ml="2px"
+            variant="text"
+            color={getOrderColor(order.state.toLowerCase())}
+            style={{ textTransform: "capitalize" }}
+          >
+            {order.state.toLowerCase()}
+          </Text>
+        </Flex>
+      </Flex>
+      <Flex p={2} width="100%" alignItems="center">
+        <Flex width="50%">
+          {artworkImage}
+          <Flex flexDirection="column" ml={1}>
+            {!orderIsInactive && (
+              <Link
+                href={`/orders/${order.internalID}/status`}
+                underlineBehavior="hover"
+              >
+                <Text variant="text">{artwork?.artist_names}</Text>
+              </Link>
+            )}
+            {orderIsInactive && (
+              <Text variant="text">{artwork?.artist_names}</Text>
+            )}
+            <Text variant="text" color="black60">
+              {artwork?.title}
+            </Text>
+          </Flex>
+        </Flex>
+        <Flex width="50%">
+          <Avatar size="xs" src={partnerImageUrl} initials={partnerInitials} />
+          <Flex flexDirection="column" ml={1}>
+            <Text variant="text">{partnerName}</Text>
+            <Text variant="text" color="black60">
+              {artwork?.shippingOrigin &&
+                artwork?.shippingOrigin.replace(/, US/g, "")}
```

Comment:
🙈

Reply:
This was in Lily's code. Not sure what it was but I thought it was intentional not to show country code when shipping origin is `US`

Reply:
Yeah looks like so. Curious if there is more structured way to do this. Non-blocking tho.